nalito223/hotel
# Feature/accessibility

### Summary: 
This branch completes the accessibility rubric for this project. The log in page has been commented out so it doesn't interfere with the Lighthouse audit. 

### How to test: 
Please take a look at each of the components below to see if they are functioning as expected:

- Event listener in `scripts.js` makes all clickable objects actionable using `enter` 
- All relevant elements are now in the natural tab path
- Dynamic aria state of `selected` added to the "My bookings" and "Book a room" header buttons
- Semantic elements applied in `index.html`
- Lighthouse audit at 100% passing 
- Labels added to any actionable element in `index.html` and any `innerHTML` used in `scripts.js`
